src/main/java/org/sonarsource/scanner/cli/Cli.java

```diff
@@ -33,6 +34,7 @@
   private final Logs logger;
 
   public Cli(Exit exit, Logs logger) {
+      System.out.println("hello");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.scanner.cli%3Asonar-scanner-cli&branch=feature%2Fjanos%2Fjunk1&issues=AWFQYVNasS4rYBZqE3DW&open=AWFQYVNasS4rYBZqE3DW)

[ISSUE_KEY]: AWFQYVNasS4rYBZqE3DW